### PR TITLE
Fix technical debt issues

### PR DESCRIPTION
**Pull request by tlorent**
Fix the ISO format in JSON and place all time and/or date conversion in the service file.

### CODE REVIEW

File: services/convert-date-time/convert-date-time.js [hunk 1]
```diff
@@ -6,4 +6,6 @@ const convertDateTime = (date, time) => {
   return dateToISO;
 };
 
-export default convertDateTime;
+const reformatTime = (time) => new Date(time).toLocaleTimeString('nl-NL', { hour: '2-digit', minute: '2-digit' });
```
Comment (boris-arkenaar) [inline services/convert-date-time/convert-date-time.js]: Probably better to rename the file to `date-time.js` now?

File: services/convert-date-time/convert-date-time.js [hunk 2]
```diff
@@ -6,4 +6,6 @@ const convertDateTime = (date, time) => {
...
+
+export { convertDateTime, reformatTime };
```
Comment (boris-arkenaar) [inline services/convert-date-time/convert-date-time.js]: You can define each export directly:
```
export const ... = () => ...;
```

File: components/timesheet/Timesheet.jsx [hunk 3]
```diff
@@ -1,28 +1,28 @@
...
-  employer, startTime, stopTime, onReformatTime
+  employer, startTime, stopTime
 }) => (
   <div className="timesheet">
     <p className="timesheet__title">
       {employer}
     </p>
     <p className="timesheet__time">
-      {onReformatTime(startTime)}
+      {reformatTime(startTime)}
```
Comment (aronmartin) [inline components/timesheet/Timesheet.jsx]:
```
{`${reformatTime(startTime)} - ${reformatTime(stopTime)}`}
```

Comment (tlorent) [reply components/timesheet/Timesheet.jsx]: Of course, forgot that again, thanks!

**Pull request merged**